docs/_docs/reference/other-new-features/generalized-method-syntax.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: doc-page
+title: "Generalized Method Syntax"
+movedTo: https://docs.scala-lang.org/scala3/reference/other-new-features/generalized-method-syntax.html
+---
+
+The inclusion of using clauses is not the only way in which methods have been updated, type parameter clauses are now allowed in any number and at any position.
+
+## Syntax Changes
+
+### In Scala 2
+
+The old syntax only allowed zero or one type parameter clause, followed by any number of term clauses, optionnally followed by an implicit clause:
+
+```scala
+def foo[T, U](x: T)(y: U)(z: Int, s: String)(a: Array[T])(implicit List[U])
+```
+
+### In Scala 3
+
+The new syntax allows any number of type, term and using clause, in any order, optionnaly followed by an implicit clause:
```

Comment:
The new syntax should be marked as experimental first. It should be available under a compiler option, e.g. `-Yinterleaved-clauses`.

Reply:
isn't an experimental language import more idiomatic?

Reply:
It shouldn't be needing a special flag, IMO. The feature should just be available if `experimental` is imported.